lib/pbench/server/api/resources/upload_api.py

```diff
@@ -141,6 +142,13 @@ def put(self, filename: str):
             tar_full_path = self.temporary / filename
             md5_full_path = self.temporary / f"{filename}.md5"
             bytes_received = 0
+            usage = shutil.disk_usage(tar_full_path.parent)
+            self.logger.info(
+                "{} UPLOAD (pre): {}% full, {} bytes remaining",
+                tar_full_path.name,
+                float(usage.used) / float(usage.total) * 100.0,
+                usage.free,
+            )
```

Comment:
Are you sure that you want this at `info` level (and not, say, `debug` level), and shouldn't the call to `disk_usage()` be under a conditional [`isEnabledFor()`](https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor)?

Ditto lines 289 and 288, and possibly `server/bin/pbench-backup-tarballs` line 385 and 384.

Reply:
Effectively, `debug` level is invisible unless we do something to change the default logging level within the containerized server. We have no mechanism for that, and I decided not to even worry about it. I was curious whether anyone would comment here, but since I have two (three?) approvals, and I think the basic bug I'm fixing may be breaking other PRs, I'd rather get this in.